scripts/netbios-interfaces.nse

```diff
@@ -0,0 +1,68 @@
+local netbios = require "netbios"
+local nmap = require "nmap"
+local string = require "string"
+local tab = require "tab"
+
+description = [[
+Attempts to retrieve the target's network interfaces.
+It is very useful for finding pathes to isolated networks.
+]]
```

Comment:
The description should be more substantial. As an example, the current version does not capture what technique the script is using so users cannot make informed decisions whether to run the script or not.

Reply:
\+